src/main/java/com/epam/lowcost/DAO/UserDAOImpl.java

```diff
@@ -14,9 +17,10 @@
 public class UserDAOImpl implements UserDAO {
 
     private DataSource dataSource;
+    private JdbcTemplate jdbcTemplate;
```

Comment:
Why did you made a separate bean from jdbcTemplate and used it as class field instead of use it as local variable?

Reply:
That was first thing i've thinked of. Changed onto another realization without creating new bean.